pkg/validations/assert.go

```diff
@@ -59,8 +64,8 @@ func (a *convertAssertAnnsToValidations) Visit(node yamlmeta.Node) error {
 // If any value in the annotation is not well-formed, it returns an error.
 func NewValidationFromValidationAnnotation(annotation template.NodeAnnotation) (*NodeValidation, error) {
 	var rules []rule
-
-	if len(annotation.Args) == 0 {
+	// TODO: what makes a malformed annotation??
```

Comment:
Is this a malformed annotation? `#@schema/validation max_len=3, min_len=5`

Reply:
I'd say it's well-formed because it's syntactically correct.

However, it just won't work in practice. I have an assumption that library authors will test their business... and will discover that there's no set of data values that can satisfy all these rules.